common/src/Admin.php

```diff
@@ -88,6 +88,10 @@ public function options_page() {
             }
 
             if (!$errors) {
+                $nonce = $_REQUEST['_wpnonce'];
+                if ( ! wp_verify_nonce( $nonce, 'submit_trustmark_settings' ) ) {
```

Comment:
A better name for the nonce would be:

~~~php
$this->plugin->getOptionName('options_nonce')
~~~

This identifies the plugin. The current name is a bit generic.